packages/kbn-babel-preset/common_preset.js

```diff
@@ -40,6 +40,9 @@ module.exports = () => ({
         // Need this since we are using TypeScript 3.9+
         require.resolve('@babel/plugin-proposal-private-methods'),
 
+        // Needed for ??= usage in our vega-* dependencies
+        require.resolve('@babel/plugin-transform-logical-assignment-operators'),
```

Comment:
Was this needed?  https://babeljs.io/docs/babel-plugin-transform-logical-assignment-operators notes this is already included in preset-env.

https://github.com/elastic/kibana/blob/8e21442caa8e93ab85aed64c102f972985f96ab3/packages/kbn-babel-preset/webpack_preset.js#L17

Reply:
Maybe it's not needed, but this is what I found was necessary to get it compile locally. 

In this file, are we referencing this preset (kbn-babel-preset) that references @babel/preset-env?

Reply:
Ah, browserslist is telling babel-preset not to use the transform and webpack 4 doesn't understand it.

It is loaded a few lines below.